howso/engine/trainee.py

```diff
@@ -962,6 +967,76 @@ def auto_analyze(self) -> None:
             self.client.auto_analyze(self.id)
         else:
             raise ValueError("Client must have the 'auto_analyze' method.")
+    
+    def get_auto_ablate_params(self):
+        """
+        Get parameters set by :meth:`set_auto_ablate_params`.
+        """
+        if isinstance(self.client, AbstractHowsoClient):
+            self.client.get_auto_ablate_params()
+        else:
+            raise ValueError("Client must have the 'get_auto_ablate_params' method.")
+    
+    def set_auto_ablate_params(
+        self,
+        auto_ablate_enabled: bool = False,
+        auto_ablate_weight_feature: str = ".case_weight",
+        minimum_model_size: int = 1_000,
+        influence_weight_entropy_threshold: float = 0.6,
+        exact_prediction_features: Optional[List[str]] = None,
+        residual_prediction_features: Optional[List[str]] = None,
+        tolerance_prediction_threshold_map: Optional[Dict[str, Tuple[float, float]]] = None,
+        relative_prediction_threshold_map: Optional[Dict[str, float]] = None,
+        conviction_lower_threshold: Optional[float] = None,
+        conviction_upper_threshold: Optional[float] = None,
+        **kwargs
+    ):
+        """
+        Set trainee parameters for auto ablation.
+
+        .. note::
+            Auto-ablation is experimental and the API may change without deprecation.
+
+        Parameters
+        ----------
+        auto_ablate_enabled : bool, default False
+            When True, the :meth:`train` method will ablate cases that meet the set criteria.
+        auto_ablate_weight_feature : str, default ".case_weight"
+            The weight feature that should be accumulated to when cases are ablated.
+        minimum_model_size : int, default 1,000
+            The threshold ofr the minimum number of cases at which the model should auto-ablate.
+        influence_weight_entropy_threshold : float, default 0.6
+            The influence weight entropy quantile that a case must be beneath in order to be trained.
+        exact_prediction_features : Optional[List[str]], optional
+            For each of the features specified, will ablate a case if the prediction matches exactly.
+        residual_prediction_features : Optional[List[str]], optional
+            For each of the features specified, will ablate a case if abs(prediction - case value) / prediction <= feature residual.
```

Comment:
If you shorten these lines (or break them across several lines) in other places, please do it here as well.